Tasks/Sepseleva/Task7/data.py

```diff
@@ -0,0 +1,67 @@
+import copy
+from decimal import Decimal
+repo = {
+    12341:{"name":"Harry Potter","author":"J.K. Rowling", "price": '36.99', "category":"Fantasy", "number":5},
```

Comment:
Harry Potter and what exactly?))

Reply:
Haha) Shame on me :) , Actually, I couldn't choose one and didn't want to add all of them here) Let it be "Harry Potter and the Deathly Hallows: Part 1")